api/pkg/transformer/feast/validator.go

```diff
@@ -0,0 +1,92 @@
+package feast
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/feast-dev/feast/sdk/go/protos/feast/core"
+	"github.com/pkg/errors"
+
+	"github.com/gojek/merlin/pkg/transformer"
+)
+
+// ValidateTransformerConfig validate transformer config by checking the presence of entity and features in feast core
+func ValidateTransformerConfig(ctx context.Context, coreClient core.CoreServiceClient, trfCfg *transformer.StandardTransformerConfig) error {
+	if trfCfg.TransformerConfig == nil {
+		return NewValidationError("transformerConfig is empty")
+	}
+
+	if len(trfCfg.TransformerConfig.Feast) == 0 {
+		return NewValidationError("feature retrieval config is empty")
+	}
+
+	req := &core.ListEntitiesRequest{}
```

Comment:
nitpick: we can specify filter options: `https://github.com/feast-dev/feast/blob/v0.8.2/sdk/go/protos/feast/core/CoreService.pb.go#L2331-L2343`

Reply:
I want to get all entities in one call, thus not using filter